spec/rspec/core/formatters/failure_list_formatter_spec.rb

```diff
@@ -4,15 +4,42 @@ module RSpec::Core::Formatters
       output = run_example_specs_with_formatter('failures')
+      output.gsub!(/`.*?'$/, "`...'") # JRuby dumps block nesting slightly differently
       expect(output).to eq(<<-EOS.gsub(/^\s+\|/, ''))
-        |./spec/rspec/core/resources/formatter_specs.rb:4:is marked as pending but passes
-        |./spec/rspec/core/resources/formatter_specs.rb:36:fails
-        |./spec/rspec/core/resources/formatter_specs.rb:40:fails twice
-        |./spec/rspec/core/resources/formatter_specs.rb:47:fails with a backtrace that has no file
-        |./spec/rspec/core/resources/formatter_specs.rb:53:fails with a backtrace containing an erb file
-        |./spec/rspec/core/resources/formatter_specs.rb:71:raises
+        |./spec/rspec/core/resources/formatter_specs.rb:4:E:Expected example to fail since it is pending, but it passed.
+        |./spec/rspec/core/resources/formatter_specs.rb:37:E:expected: 2 got: 1 (compared using ==)
+        |./spec/rspec/core/resources/formatter_specs.rb:37:I:in `...'
+        |./spec/support/formatter_support.rb:41:I:in `...'
+        |./spec/support/formatter_support.rb:3:I:in `...'
+        |./spec/rspec/core/formatters/failure_list_formatter_spec.rb:8:I:in `...'
+        |./spec/support/sandboxing.rb:16:I:in `...'
+        |./spec/support/sandboxing.rb:7:I:in `...'
+        |./spec/rspec/core/resources/formatter_specs.rb:41:E:expected: 2 got: 1 (compared using ==)
+        |./spec/rspec/core/resources/formatter_specs.rb:41:I:in `...'
+        |./spec/support/formatter_support.rb:41:I:in `...'
+        |./spec/support/formatter_support.rb:3:I:in `...'
+        |./spec/rspec/core/formatters/failure_list_formatter_spec.rb:8:I:in `...'
+        |./spec/support/sandboxing.rb:16:I:in `...'
+        |./spec/support/sandboxing.rb:7:I:in `...'
+        |./spec/rspec/core/resources/formatter_specs.rb:42:E:expected: 4 got: 3 (compared using ==)
+        |./spec/rspec/core/resources/formatter_specs.rb:42:I:in `...'
+        |./spec/support/formatter_support.rb:41:I:in `...'
+        |./spec/support/formatter_support.rb:3:I:in `...'
+        |./spec/rspec/core/formatters/failure_list_formatter_spec.rb:8:I:in `...'
+        |./spec/support/sandboxing.rb:16:I:in `...'
+        |./spec/support/sandboxing.rb:7:I:in `...'
+        |./spec/rspec/core/resources/formatter_specs.rb:50:E:foo
+        |./spec/rspec/core/resources/formatter_specs.rb:50:I:in `...'
+        |./spec/support/formatter_support.rb:41:I:in `...'
+        |./spec/support/formatter_support.rb:3:I:in `...'
+        |./spec/rspec/core/formatters/failure_list_formatter_spec.rb:8:I:in `...'
+        |./spec/support/sandboxing.rb:16:I:in `...'
+        |./spec/support/sandboxing.rb:7:I:in `...'
+        |/foo.html.erb:1:E:Exception
+        |/foo.html.erb:1:I:in `<main>': foo (RuntimeError)
+        |./spec/rspec/core/resources/formatter_specs.rb:71:E:boom
```

Comment:
This massively changes the reason for the formatter, so I'm unsure thats what we want. I'd prefer a much simpler version of this PR that just swaps out the message of the failure from example description to a string from an exception/exceptions (single line per test).